tsconfig.json

```
@@ -16,10 +16,14 @@
     "noEmit": true,
     "jsx": "react-jsx",
     "types": ["vite/client", "vite-plugin-pwa/client"],
+    "baseUrl": ".",
     "paths": {
+      "@core/*": ["src/*"],
```

Do you need to call it core? I would prefer something like this. 
```suggestion
      "@/*": ["./src/*"],
```

---

Done